source/simulator/helper_functions.cc

```diff
@@ -1554,10 +1580,7 @@ namespace aspect
     Assert (reaction_time_step_size > 0,
             ExcMessage("Reaction time step must be greater than 0."));
 
-    pcout << "   Solving composition reactions in "
-          << number_of_reaction_steps
-          << " substep(s)."
-          << std::endl;
+    pcout << "   Solving composition reactions ";
```

Comment:
This will likely not show on the screen until you complete the line. You might have to add a `std::flush` to make it show.

Do we have other places where we place preliminary output on a line and later complete it? I think that sometimes we end the preliminary text with `...` but I'm not sure whether we're consistent.